desktop/apps/collect/routes.coffee

```diff
@@ -1,19 +1,26 @@
 qs = require 'qs'
 Q = require 'bluebird-q'
 _ = require 'underscore'
+{ capitalize } = require 'underscore.string'
 Backbone = require 'backbone'
 JSONPage = require '../../components/json_page'
 page = new JSONPage name: 'browse-categories'
 GeocodedCities = require '../../collections/geocoded_cities'
+{ COLLECT_PAGE_TITLES_URL } = require('sharify').data
+getCollectPageTitle = require '../../components/commercial_filter/page_title'
 
 @index = (req, res, next) ->
   geocodedCities = new GeocodedCities()
+  collectPageTitleFilters = new Backbone.Collection()
   Q.all([
     geocodedCities.fetch()
+    collectPageTitleFilters.fetch(url: COLLECT_PAGE_TITLES_URL)
     page.get()
-  ]).spread (geocodedCities, pageData) ->
+  ]).spread (geocodedCities, pageTitleFilters, pageData) ->
```

Comment:
interesting. i wasn't aware of this [spread API](http://bluebirdjs.com/docs/api/spread.html) on promises.

Reply:
Yea, its nice!